README.md

```diff
@@ -1,133 +1,111 @@
 
-You should see the below output - if not then head back to [installation instructions](documentation/example-file-usage/example-file-prerequistes.md)
-```
-Usage: ankerctl.py [OPTIONS] COMMAND [ARGS]...
-
-Options:
-  -k, --insecure  Disable TLS certificate validation
-  -v, --verbose   Increase verbosity
-  -q, --quiet     Decrease verbosity
-  -h, --help      Show this message and exit.
-
-Commands:
-  config  View and update configuration
-  http    Low-level http api access
-  mqtt    Low-level mqtt api access
-  pppp    Low-level pppp api access
-```
+   docker:
 
-Before you can use ankerctl, you need to import the configuration.
+   ```sh
+   docker compose up
+   ```
 
-Support for logging in with username and password is not yet supported, but an
-`auth_token` can be imported from the saved credentials found in `login.json` in
-Ankermake Slicer. See `ankerctl.py config import -h` for details:
+   python:
 
-```
-Usage: ankerctl.py config import [OPTIONS] path/to/login.json
+   ```sh
+   ./ankerctl.py webserver run
+   ```
 
-  Import printer and account information from login.json
+2. Navigate to [http://localhost:4470](http://localhost:4470) in your browser of choice on the same computer the webserver is running on. You’ll be prompted to upload your `login.json` file and the given the default path it should be found in your corresponding Operating System. Once the `login.json` has been uploaded, the page will refresh and the web interface is usable. To access it from other devices (must be on the same network), replace “localhost” with the IP address of the computer running the web server.
 
-  When run without filename, attempt to auto-detect login.json in default
-  install location
+### CLI Utilities (Python Scripts)
 
-Options:
-  -h, --help  Show this message and exit.
-```
+1. Import your AnkerMake account data by opening a terminal window in the folder you placed ankerctl in and running the following command:
 
-On Windows and MacOS, the default location of `login.json` will be tried if no
-filename is specified. Otherwise, you can specify the file path for
-`login.json`. Example for linux:
+   ```sh
+   ./ankerctl.py config import [OPTIONS] path/to/login.json 
+   ```
 
-```sh
-ankerctl.py config import ~/.wine/drive_c/users/username/AppData/Local/AnkerMake/AnkerMake_64bit_fp/login.json
-```
+   When run without filename on Windows and MacOS, the default location of `login.json` will be tried if no filename is specified. Otherwise, you can specify the file path for `login.json`. Example for Linux:
+   ```sh
+   ./ankerctl.py config import ~/.wine/drive_c/users/username/AppData/Local/AnkerMake/AnkerMake_64bit_fp/login.json
+   ```
+   MacOS
+   ```sh
+   ./ankerctl.py config import $HOME/Library/Application\ Support/AnkerMake/AnkerMake_64bit_fp/login.json
+   ```
+   Windows
+   ```sh
+   python3 ankerctl.py config import %APPDATA%\AnkerMake\AnkerMake_64bit_fp\login.json
```

Comment:
MacOS and Windows shouldn't need these paths, right?

I'm *all* for having these examples, but we might want to mention that `config import` might be enough on its own?